reactive/kubernetes_master.py

```diff
@@ -2961,8 +2984,7 @@ def configure_hacluster():
         add_service_to_hacluster(service, daemon)
 
     # get a new cert
-    if (is_state('certificates.available') and
-            is_state('kube-api-endpoint.available')):
+    if all_flags_set("certificates.available", "kube-api-endpoint.available"):
```

Comment:
The early check-and-return does make these additional checks superfluous, but I've no objection to leaving them in anyway.